modules/legacy-geo/src/test/java/org/elasticsearch/legacygeo/GeoJsonShapeParserTests.java

```diff
@@ -390,7 +391,7 @@ public void testParse3DPolygon() throws IOException, ParseException {
 
         LinearRing shell = GEOMETRY_FACTORY.createLinearRing(shellCoordinates.toArray(new Coordinate[shellCoordinates.size()]));
         Polygon expected = GEOMETRY_FACTORY.createPolygon(shell, null);
-        final IndexVersion version = IndexVersionUtils.randomCompatibleVersion(random());
+        final IndexVersion version = IndexVersionUtils.randomPreviousCompatibleVersion(random(), IndexVersions.V_8_0_0);
```

Comment:
I'm curious why this doesn't fail, MININUM_COMPATIBLE should be at 8.0.0 but we should work with 7.9.0 here. Just a remark though

Reply:
I believe for all of these, I took what we have in main, that was adjusted with the 9x bump.